src/styles/reviews.scss

```diff
@@ -0,0 +1,43 @@
+@import 'variables';
+
+.reviews {
+  display: flex;
+  box-sizing: border-box;
+  margin: 0 auto 24px;
+  width: 166px;
+  height: 16px;
+  text-align: center;
+  justify-content: space-between;
+  padding: 2px 0 0 2px;
```

Comment:
The 'width' and 'height' properties should be placed before 'text-align' and 'justify-content' to adhere to the box model order in the CSS properties.